scripts/SplitStockholm.jl

```diff
@@ -35,26 +40,36 @@ end
 const Args = parse_commandline()
 
 function main(input)
-	infh = GZip.open(input)
-	lines = []
-	id = "no_accessionumber"
-	for line in eachline(infh)
-		if length(line) > 7 && line[1:7] == "#=GF AC"
-			id = get_n_words(line, 3)[3]
-		end
-		push!(lines, line)
-		if line == "//\n"
-			filename = joinpath(Args["path"], string(id, ".gz"))
-			outfh = GZip.open(filename, "w")
-			for l in lines
-				write(outfh, string(l))
-			end
-			close(outfh)
-			id = "no_accessionumber"
-			empty!(lines)
-		end
-	end
-	close(infh)
+    infh = GZip.open(input)
+    lines = []
+    id = "no_accessionumber"
+
+    if Args["progress"]:
+        thresh = filesize(infh)
+        prog = ProgressThresh(thresh, "Bytes read:")
+    end
+
+    while !eof(infh)
+        line = readline(infh)
+        if length(line) > 7 && line[1:7] == "#=GF AC"
+            id = get_n_words(line, 3)[3]
+        end
+        push!(lines, line)
+        if line == "//\n"
+            filename = joinpath(Args["path"], string(id, ".gz"))
+            outfh = GZip.open(filename, "w")
+            for l in lines
+                write(outfh, string(l))
+            end
+            close(outfh)
+            id = "no_accessionumber"
+            empty!(lines)
+            if Args["progress"]:
+                update!(prog, prog.value + filesize(outfh))
```

Comment:
I'm getting this error: `type ProgressThresh has no field value`

Maybe you can define a `value` variable initialized in 0, and updated it with `value += filesize(filename) ` before calling `update!`